index.php

```diff
@@ -127,19 +131,91 @@
                 <div class="osekai__3col_col2">
                     <section class="osekai__panel">
                         <div class="osekai__panel-header">
-                            <p>A (very old) user panel</p>
+                            <p>Comments</p>
                         </div>
-                        <div class="osekai__panel-inner">
-                            <div class="osekai__section-header">
-                                <div class="osekai__section-header-left">
-                                    <h2>(not so sick) Sick user panel</h2>
+                        <div class="osekai__panel-inner comments__section">
+                            <div class="comments__post">
+                                <div class="comments__post-left">
+                                    <img src="https://a.ppy.sh/1" class="comments__pfp">
                                 </div>
-                                <div class="osekai__section-header-right">
-                                    <h3>gotta love it</h3>
-                                    <p>right?... right???</p>
+                                <div class="comments__post-right">
+                                    <div class="comments__post-upper">
+                                        <textarea rows="1" placeholder="Post a comment!"></textarea>
+                                    </div>
+                                    <div class="comments__post-lower">
+                                        <div class="comments__post-lower-right">
+                                            <p>osekai supports bbcode!</p>
+                                            <div class="comments__post-button"><i class="fas fa-plane"></i> Post</div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="osekai__divider"></div>
+                            <div class="comments__list">
+                                <div class="comments__comment">
+                                    <div class="comments__comment-left">
+                                        <img src="https://a.ppy.sh/1" class="comments__pfp">
+                                        <div class="comments__comment-votes">
+                                            +22
+                                        </div>
+                                    </div>
+                                    <div class="comments__comment-right">
+                                        <div class="comments__comment-content">
+                                            <div class="comments__comment-content-username">
+                                                <p class="comments__username">Username</p>

```

Comment:
Consider as a link to the osekai-profile of said user.

Reply:
have done (please stop looking at the html on index.php it's not representative of what's on the page, it was just to test css...)